### DISCUSSION TITLE
Windows Quick Launch toolbar?

### DISCUSSION THREAD

**Discussion by lovetolearncode** [ramensoftware/windhawk#178] [Ideas]
Is there a MOD to get back that older Windows Quick Launch toolbar? It is a really nice toolbar.
Thanks!

**Comment by BugFixNo1** [comment]
iI would also find such a mod excellent and would appreciate the work

**Comment by Anixx** [comment]
There is: https://windhawk.net/mods/win10-taskbar-on-win11
